pkg/sysfsnet/interfaces_test.go

```diff
@@ -45,4 +50,9 @@ func TestInterfaces(t *testing.T) {
 	if !reflect.DeepEqual(want, got) {
 		t.Fatalf("want MACs=%v, got MACs=%v", want, got)
 	}
+
+	// Ensure non-directory entry is ignored
+	if _, exists := got[""]; exists {
```

Comment:
I think `got[""]` is too broad. Can you add some test string to the `bonding_masters` file and then make sure they aren't present in `got`.